sympy/printing/tests/test_latex.py

```diff
@@ -538,6 +538,11 @@ def test_latex_sequences():
     assert latex(SeqMul(s5, s6)) == latex_str
 
 
+def test_latex_FourierSeries():
+    latex_str = r'2 \sin{\left (x \right )} - \sin{\left (2 x \right )} + \frac{2}{3} \sin{\left (3 x \right )}\ldots'
```

Comment:
The normal print methods have `+ ...` at the end rather than `...`, the latex should probably do the same